package.json

```diff
@@ -130,6 +130,7 @@
     "react-input-autosize": "1.1.0",
     "react-intl": "2.2.3",
     "react-joyride": "1.10.1",
+    "react-notification-system": "0.2.14",
```

Comment:
Uhm... it seems to me this module is not reactish at all if we use the NotificationSystem component. Basically it's the most imperative component I have seen til now. It would be better to use the NotificationsContainer directly, in my opinion, or use https://github.com/gor181/react-notification-system-redux that is a wrapper hiding the imperative stuff